Documentation/Advanced.md

```diff
@@ -67,7 +67,7 @@ struct AuthorizedRequest: Request {
         return Result(false)
       default:
         let error = NSError.error("Unexpected response body!")
-        return Result(error: .InvalidJSONResponse(error))
+        return Result(error: .DeserializationError(error))
```

Comment:
Good catch, since `.InvalidJSONReponse` doesn't exist anymore. I'd prefer to call this a ` .ParseError` now since that's the step were in here. Thoughts?